packages/react-native/React/Views/RCTShadowView.m

```diff
@@ -182,6 +182,25 @@ - (CGRect)measureLayoutRelativeToAncestor:(RCTShadowView *)ancestor
   return (CGRect){offset, self.layoutMetrics.frame.size};
 }
 
+// [macOS
+- (RCTShadowView *)ancestorSharedWithShadowView:(RCTShadowView *)shadowView
+{
+  // TODO: This can be optimized by climbing up both hierarchies at the same time
```

Comment:
Still relevant?

Reply:
Yes, we use it [here](https://github.com/microsoft/react-native-macos/pull/2143/files#diff-eeec5591dac416878467a1005d3ee9f0ebe76321b9ecfc92c22c4409d60692c2R537).

Reply:
I meant, can we remove the TODO?

Reply:
The TODO is a separate implementation detail and it's more of an open question than anything else. We can probably reword the TODO, but it's not a high-priority TODO.